students/zach_thomson/lesson01/inventory_management/electric_appliances_class.py

```diff
@@ -0,0 +1,21 @@
+'''module that creates an electric appliances class'''
+
+# Electric appliances class
+from inventory_management.inventory_class import Inventory
```

Comment:
Change this line back to just calling _inventory_class_, same for _furniture_class.py_:

```
from inventory_class import Inventory
```